src/backend/distributed/master/master_delete_protocol.c

```diff
@@ -216,25 +219,24 @@ master_drop_all_shards(PG_FUNCTION_ARGS)
 
 
 /*
- * master_drop_sequences attempts to drop a list of sequences on a specified
- * node. The "IF EXISTS" clause is used to permit dropping sequences even if
- * they may not exist. Returns true on success, false on failure.
+ * master_drop_sequences attempts to drop a list of sequences on all nodes.
+ * The "IF EXISTS" clause is used to permit dropping sequences even if
+ * they may not exist.
  */
 Datum
 master_drop_sequences(PG_FUNCTION_ARGS)
 {
 	ArrayType *sequenceNamesArray = PG_GETARG_ARRAYTYPE_P(0);
-	text *nodeText = PG_GETARG_TEXT_P(1);
-	int64 nodePort = PG_GETARG_INT64(2);
-	bool dropSuccessful = false;
-	char *nodeName = TextDatumGetCString(nodeText);
-
 	ArrayIterator sequenceIterator = NULL;
 	Datum sequenceText = 0;
 	bool isNull = false;
-
 	StringInfo dropSeqCommand = makeStringInfo();
 
+	if (!EnableDDLPropagation)
```

Comment:
Well, in mx and in general, unless if you really know what you are doing, disabling ddl propagation is a very bad idea and doing ddl-related changes with ddl propagation of is almost always going to break things. 

I will look into the crash.